cmd/vm_stop.go

```diff
@@ -23,30 +22,34 @@ var vmStopCmd = &cobra.Command{
 			return err
 		}
 
-		errs := []error{}
+		tasks := make([]task, 0, len(args))
 		for _, v := range args {
 			if !force {
 				if !askQuestion(fmt.Sprintf("Are you sure you want to stop virtual machine %q?", v)) {
-					return nil
+					continue
 				}
 			}
 
-			if err := stopVirtualMachine(v); err != nil {
-				errs = append(errs, fmt.Errorf("could not stop %q: %s", v, err))
+			vm, err := getVirtualMachineByNameOrID(v)
+			if err != nil {
+				return err
 			}
-		}
+			state := (string)(egoscale.VirtualMachineRunning)
```

Comment:
Here again, the variable could be renamed `runningState`.

Reply:
same as above